### PR TITLE
Selinux FAQ

### PR DESCRIPTION
**Pull request by renzo38**
Command to run on SELinux system enabled

### CODE REVIEW

File: docs/en/users/07_Frequently_Asked_Questions.md [hunk 1]
```diff
@@ -43,4 +43,24 @@ Since [1.8.0](https://github.com/FreshRSS/FreshRSS/releases/tag/1.8.0) release,
...
+
+ - FreshRSS/data
+ - FreshRSS/data/cache
+ - FreshRSS/data/favicons
+ - FreshRSS/data/users
+
+The following commands should be executed as root :
+
+```sh
+semanage fcontext -a -t httpd_sys_rw_content_t '/usr/share/FreshRSS/data
```
Comment (aledeg) [inline docs/en/users/07_Frequently_Asked_Questions.md]: I guess it's missing a quote at the end of the line. Same goes in the other file

**Review by aledeg — CHANGES_REQUESTED** [review]
I think there is typo. Anyway, thank you for your contribution.

**Comment by renzo38** [comment]
I added the missing quotes, sorry

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/issues/1835#issuecomment-373907931

**Comment by aledeg** [comment]
At first, I didn't see that the quotes you've corrected where missing :). Good catch. I was talking at the end of the line I've pointed. I should have:
```
semanage fcontext -a -t httpd_sys_rw_content_t '/usr/share/FreshRSS/data'
```
instead of 
```
semanage fcontext -a -t httpd_sys_rw_content_t '/usr/share/FreshRSS/data
```
Same goes in both files.

**Review by aledeg — APPROVED** [review]

**Comment by Alkarex** [comment]
Copy from https://github.com/FreshRSS/FreshRSS/issues/1835#issuecomment-373987380

> The whole `./data/` should be writeable. For instance, `./data/PubSubHubbub/` was forgotten.

Hence the following suggestion:
```sh
semanage fcontext -a -t httpd_sys_rw_content_t '/usr/share/FreshRSS/data(/.*)?'
restorecon -Rv /usr/share/FreshRSS/data
```

**Comment by Alkarex** [comment]
@renzo38 Did you maybe close this issue by mistake? :-)

**Comment by renzo38** [comment]
I closed this by mistake, sorry, doing several things together lead to this, sorry...

**Comment by Alkarex** [comment]
@renzo38 Did you have a change to consider my suggestion, a bit higher up?
https://github.com/FreshRSS/FreshRSS/pull/1837#issuecomment-373992230

**Comment by renzo38** [comment]
So, I will modify the command to set the rights on the whole data sub-dirs.

**Comment by Alkarex** [comment]
@renzo38 Is there a reason why you do not like my one-line suggestion https://github.com/FreshRSS/FreshRSS/pull/1837#issuecomment-373992230 ?
Everything under `./data/` is supposed to be writeable, otherwise it is not located there.

**Comment by renzo38** [comment]
No, no it's ok, I added the line at the end of the Readme, I can remove the previous lines, no problem.

**Pull request merged**